tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1234,3 +1236,44 @@ def test_email_of_the_user_should_be_copied(session, auth_source_ipa, ipa_data,
     with session:
         user_value = session.user.read(ipa_data['user_ipa'])
         assert user_value['user']['mail'] == result
+
+
+@tier2
+def test_deleted_idm_user_should_not_be_able_to_login(auth_source_ipa, ldap_tear_down):
+    """After deleting a user in IDM, user should not be able to login into satellite
+
+    :id: 18ad0526-e083-11ea-b1ad-4ceb42ab8dbc
+
+    :steps:
+        1. Create a new auth source with onthefly enabled
+        2. Create a new user in IDM and assigning a group to it.
+        3. Login to satellite to create the user
+        4. Delete the user from IDM
+        5. Try login to the satellite from the user
+
+    :expectedresults: User login fails
+    """
+    run_command(
+        cmd=f"echo {settings.ipa.password_ipa} | kinit admin", hostname=settings.ipa.hostname_ipa
+    )
```

Comment:
Can we add the assert statement to check the command execution status, if some kind of environmental issue happens then we can easily track that?
assert result.return_code == 0

Reply:
added.